ix-dev/community/windmill/ix_values.yaml

```diff
@@ -0,0 +1,29 @@
+images:
+  image:
+    repository: ghcr.io/windmill-labs/windmill
+    tag: "1.485.3"
+  postgres_16_image:
```

Comment:
lets start with pg 17

Reply:
Windmill themselves use 16, which is why

Reply:
afaik 17 is fully backwards compatible. starting with latest will remove the need to upgrade the db later.

Reply:
```suggestion
  postgres_upgrade_image:
    repository: ixsystems/postgres-upgrade
    tag: 1.0.1
  postgres_16_image:
```
Please also include this